docs/tracked-resource-beyond-thrid-level.md

```diff
@@ -12,22 +12,10 @@ ARM OpenAPI(swagger) specs
 
 - RPC-Put-V1-19
 
-## Output Message
-
-The tracked resource {resourceName} is beyond third level of nesting.
-
 ## Description
 
 Tracked resources must not be used beyond the third level of nesting.
```

Comment:
Should this be beyond the second level of nesting? Meaning tracked is ok for first (parent) and second level (child) resources, but not third level (grandchild). Make sure this is consistent with line 21.

```suggestion
Tracked resources must not be used beyond the second level of nesting.
```